roles/gateway/tasks/main.yml

```diff
@@ -19,6 +19,7 @@
 # Manage OpenVPN users list using secrets repo
 - include: users.yml
   tags:
+    - always
```

Comment:
I feel like I had a better explanation for this yesterday but now all I can think of is "why not?"

One can still call the users tag directly and only run those tasks so the tag isn't useless.